emr_launcher.py

```diff
@@ -18,28 +20,31 @@ def configure_log():
     logger = logging.getLogger("emr_launcher")
     logger.propagate = False
     console_handler = logging.StreamHandler()
-    formatter = jsonlogger.JsonFormatter(
-        "%(asctime)s %(name)-12s %(levelname)-8s %(message)s"
-    )
-    console_handler.setFormatter(formatter)
+    #formatter = jsonlogger.JsonFormatter(
```

Comment:
Can we put this JSON logging back in please? It's the standard format for logging on the project so that we can feed it into things like CloudWatch insights, etc.

Reply:
Fixed by https://github.com/dwp/emr-launcher/pull/2/commits/a68e359450ad073ff1bfdf0e37a567fa915c2bf2